### PR TITLE
PLNSRVCE-1350: move to openshift-pipelines 1.11 in prod (porting changes used for staging)

### PR DESCRIPTION
**Pull request by gabemontero**
This is moving our bump to 1.11 openshift pipelines in staging to prod

Aside from the basic version bump, this PR approximates the following PRs that were made to staging only:

- https://github.com/redhat-appstudio/infra-deployments/pull/2108
- https://github.com/redhat-appstudio/infra-deployments/pull/2094

Also note, the following PRs were processed at the time we went to 1.11 in staging, but are considered to already be applicable to the prod env:
- https://github.com/redhat-appstudio/infra-deployments/pull/2096
- https://github.com/redhat-appstudio/infra-deployments/pull/2102

Also NOTE/REMEMBER - the e2e's are NOT affected by this PR since the work off the developer overlay, which has been at 1.11 since the staging bump.

I will place a hold on this until all the reviewers sign off (as they contributed to the staging bump).  PLEASE - nobody hit the merge button on this one until all the reviewers sign off.

Also, if any of the reviewers fell others should review, please add them.

### CODE REVIEW

**Comment by gabemontero** [comment]
/hold

until I see all the reviewers have signed off

File: components/pipeline-service/production/base/pipelines-as-code-secret.yaml
Comment (Roming22) [inline components/pipeline-service/production/base/pipelines-as-code-secret.yaml:1]: This file probably conflicts with `components/pipeline-service/base/external-secrets/pipelines-as-code/pipelines-as-code-secret.yaml`. The external secret is already defined for staging in `components/pipeline-service/staging/base/pipelines-as-code-secret.yaml`, so it should be safe to remove the `components/pipeline-service/base/external-secrets/pipelines-as-code` folder and edit `components/pipeline-service/base/external-secrets/kustomization.yaml` to remove the reference.

File: components/pipeline-service/production/base/pipelines-as-code-secret.yaml
Comment (Roming22) [inline components/pipeline-service/production/base/pipelines-as-code-secret.yaml:13]: For production we should probably be a bit more aggressive on the refreshes. I would much prefer 5mins in case secrets needs to be rotated (which I just had to do a month ago).

File: components/pipeline-service/production/base/pipelines-as-code-secret.yaml
Comment (Roming22) [reply components/pipeline-service/production/base/pipelines-as-code-secret.yaml:1]: I ran `kustomize build` and looked at the diff between staging and prod. Looks good to me.

File: components/pipeline-service/production/base/pipelines-as-code-secret.yaml
Comment (Roming22) [reply components/pipeline-service/production/base/pipelines-as-code-secret.yaml:13]: This comment is not a blocker.

**Review by Roming22 — APPROVED** [review]
Approved

**Review by Michkov — APPROVED** [review]

**Pull request merged**
